content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,92 @@
+
+- **Added** a configuration for Galley to use new configuration processing.
+- **Fixed** Galley to stop too many gRPC pings from closing connections.
+- **Improved** Galley to avoid control plane upgrade failures.
+
+## istioctl
+
+- **Added** `istioctl kube-uninject` to remove Istio configuration from a previously injected configuration.
+- **Added** `istioctl experimental describe pod` to produce the Istio configuration affecting a Pod.
+- **Added** `istioctl experimental manifest apply` to generate and apply an Istio installation.
+- **Added** `istioctl experimental manifest diff` to show the difference betweeen two manifests.
+- **Added** `istioctl experimental manifest generate` to generate an Istio installation manifest.
+- **Added** `istioctl experimental manifest migrate` to convert Istio's `values.yaml` file into a format that suits the Istio operator.
+- **Added** `istioctl experimental manifest versions` to list all supported Istio versions.
+- **Added** `istioctl experimental profile diff Show` to show the difference between two Istio configuration profiles.
+- **Added** `istioctl experimental profile dump` to show an Istio configuration profile.
+- **Added** `istioctl experimental profile list` to list all Istio configuration profiles.
+- **Added** `istioctl experimental add-to-mesh` to add virtual machines to an existing Istio service mesh.
+- **Added** `istioctl experimental remove-from-mesh` to remove virtual machines from an existing Istio service mesh.
+- **Promoted** the `istioctl` subcommands `metrics`, `convert-ingress`, and `dashboard` to supported features.
+
+## Documentation
+
+- **Added** [Istio Deployment Models concept](/docs/concepts/deployment-models/).
+- **Added** documentation for [SDS using trustworthy JWTs](/blog/2019/trustworthy-jwt-sds/).
+- **Added** [CNI installation and upgrade instructions](/docs/setup/additional-setup/cni/).
+- **Added** the [Istio Operator installation guide](/docs/setup/install/operator/). The Operator and related docs are experimental.
+- **Added** examples for [mesh expansion](/docs/examples/mesh-expansion/).
+- **Added** diff generation for the Helm `values.yaml` file between releases.
+- **Fixed** technical inaccuracies and confusing info from the [Traffic Management concept](/docs/concepts/traffic-management).
+- **Improved** the [Getting started page](/docs/setup/getting-started/).
+- **Improved** the [Operations guide](/docs/ops/) to improve the visibility of the troubleshooting content.
+- **Improved** the [Istio glossary](/docs/reference/glossary/).
+- **Improved** the [Observability concept](/docs/concepts/observability/). 
+
+## Other
+
+- **Added** new images based on [distroless](/docs/ops/security/harden-docker-images/) base images.
+- **Improved** the Istio CNI Helm chart to have consistent versions with Istio.
+- **Improved** Improved Kubernetes Jobs behavior. The Kubernetes Jobs now exit correctly when the job manually calls the /quitquitquit endpoint.
```

Comment:
any more detail for this?  Does this mean we now support k8s jobs in the mesh?

Reply:
We have always supported K8s jobs, they are just standard pods, right? The problem was that the job would exit and the envoy would keep running